--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/mjlab_kangaroo/robots/pal_kangaroo/kangaroo_constants.py
	modified:   src/mjlab_kangaroo/tasks/kangaroo_locomotion/rough_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/
	wandb/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/src/mjlab_kangaroo/robots/pal_kangaroo/kangaroo_constants.py b/src/mjlab_kangaroo/robots/pal_kangaroo/kangaroo_constants.py
index 3b50861..f6d23c2 100644
--- a/src/mjlab_kangaroo/robots/pal_kangaroo/kangaroo_constants.py
+++ b/src/mjlab_kangaroo/robots/pal_kangaroo/kangaroo_constants.py
@@ -120,7 +120,6 @@ KANG_ARMS_ACTUATOR_CFG = ActuatorCfg(
 KANG_PELVIS_1_ACTUATOR_CFG = ActuatorCfg(
   joint_names_expr=[
     "pelvis_1_joint",
-    "pelvis_2_joint",
   ],
   effort_limit=100.0,
   armature=0.01,
@@ -240,8 +239,8 @@ for a in KANG_ARTICULATION.actuators:
 
     for n in names:
         # skip femur or knee joints
-        if re.match(r"leg_.*_femur_joint", n) or re.match(r"leg_.*_knee_joint", n):
-            continue
+        # if re.match(r"leg_.*_femur_joint", n) or re.match(r"leg_.*_knee_joint", n):
+        #     continue
 
         if n in e and n in s and s[n]:
             KANG_ACTION_SCALE[n] = 0.25 * e[n] / s[n]
diff --git a/src/mjlab_kangaroo/tasks/kangaroo_locomotion/rough_env_cfg.py b/src/mjlab_kangaroo/tasks/kangaroo_locomotion/rough_env_cfg.py
index 8c60f2d..760a0a4 100644
--- a/src/mjlab_kangaroo/tasks/kangaroo_locomotion/rough_env_cfg.py
+++ b/src/mjlab_kangaroo/tasks/kangaroo_locomotion/rough_env_cfg.py
@@ -15,7 +15,8 @@ class KangRoughEnvCfg(LocomotionVelocityEnvCfg):
     super().__post_init__()
 
     self.scene.entities = {"robot": replace(KANG_ROBOT_CFG)}
-    self.actions.joint_pos.actuator_names=[r"^leg_(left|right)_(femur|knee)_joint$"]
+    # self.actions.joint_pos.actuator_names=[r"^(pelvis|arm|leg)_.*(1|2|3|length|4|5)_joint$"]
+    # print(self.actions.joint_pos.actuator_names)
     self.actions.joint_pos.scale = KANG_ACTION_SCALE
 
     self.events.foot_friction.params["asset_cfg"].geom_names = [