--- git commit ---
6dc405a3b235f5eb9e7597a08b1e9552d333eb49


--- git status ---
On branch lorenzo_manipulation
Your branch is up to date with 'origin/lorenzo_manipulation'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/pal_mjlab/robots/pal_tiago_pro/tiago_pro_constants.py
	modified:   src/pal_mjlab/tasks/manipulation/mdp/observations.py
	modified:   src/pal_mjlab/tasks/manipulation/tiago_pro/env_cfgs.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/src/pal_mjlab/robots/pal_tiago_pro/tiago_pro_constants.py b/src/pal_mjlab/robots/pal_tiago_pro/tiago_pro_constants.py
index 1361165..d923ecc 100644
--- a/src/pal_mjlab/robots/pal_tiago_pro/tiago_pro_constants.py
+++ b/src/pal_mjlab/robots/pal_tiago_pro/tiago_pro_constants.py
@@ -160,7 +160,7 @@ for a in TIAGO_PRO_ARTICULATION.actuators:
 
   for n in names:
     if n in e and n in s and s[n]:
-      TIAGO_PRO_ACTION_SCALE[n] = 0.15 * e[n] / s[n]
+      TIAGO_PRO_ACTION_SCALE[n] = 0.25 * e[n] / s[n] if "gripper" in n else 0.1 * e[n] / s[n]
       TIAGO_PRO_ACTUATOR_NAMES += (n,)
 
 
diff --git a/src/pal_mjlab/tasks/manipulation/mdp/observations.py b/src/pal_mjlab/tasks/manipulation/mdp/observations.py
index 5ea2771..7c635c4 100644
--- a/src/pal_mjlab/tasks/manipulation/mdp/observations.py
+++ b/src/pal_mjlab/tasks/manipulation/mdp/observations.py
@@ -42,7 +42,8 @@ def object_pose_6d_in_robot_root_frame(
   obj_quat = object_orientation_in_robot_root_frame(env, command_name, asset_cfg)
   roll, pitch, yaw = euler_xyz_from_quat(obj_quat)
   obj_euler = torch.stack([roll, pitch, yaw], dim=-1)
-  return torch.cat([obj_pos, obj_euler], dim=-1)
+  #return torch.cat([obj_pos, obj_euler], dim=-1)
+  return obj_pos
 
 
 def target_position_in_robot_base_frame(
@@ -157,7 +158,8 @@ def head_camera_keypoints(
   fingertip_site_names = [s for s in robot.site_names if "fingertip" in s]
   fingertip_pos_w = robot.data.site_pos_w[:, [robot.site_names.index(name) for name in fingertip_site_names]]
   
-  keypoints_3d_w = torch.cat([corners_3d_w, fingertip_pos_w], dim=1)
+  #keypoints_3d_w = torch.cat([corners_3d_w, fingertip_pos_w], dim=1)
+  keypoints_3d_w = corners_3d_w
   
   # 3. Camera Kinematics
   cam_idx = env.sim.mj_model.camera(f"robot/{camera_name}").id
diff --git a/src/pal_mjlab/tasks/manipulation/tiago_pro/env_cfgs.py b/src/pal_mjlab/tasks/manipulation/tiago_pro/env_cfgs.py
index 1636b78..bcd2ea8 100644
--- a/src/pal_mjlab/tasks/manipulation/tiago_pro/env_cfgs.py
+++ b/src/pal_mjlab/tasks/manipulation/tiago_pro/env_cfgs.py
@@ -117,7 +117,7 @@ def lift_env_cfg(
     object_pose_range=manipulation_mdp_pal.LiftingCommandCfg.ObjectPoseRangeCfg(
       x=(0.4, 0.6),
       y=(-0.1, 0.1),
-      yaw=(-3.1415926535, 3.1415926535),
+      yaw=(-0.77, 0.77),
     ),
   )
 
@@ -179,8 +179,8 @@ def lift_env_cfg(
   #   cfg.observations["actor"].terms[name].noise = Unoise(n_min=-0.01, n_max=0.01)
 
   cfg.observations["actor"].terms["object_pose_6d"].noise = Unoise(
-    n_min=(-0.01, -0.01, -0.01, -0.05, -0.05, -0.05),
-    n_max=(0.01, 0.01, 0.01, 0.05, 0.05, 0.05),
+    n_min=(-0.01, -0.01, -0.01, ),
+    n_max=(0.01, 0.01, 0.01, ),
   )
 
   #### REWARDS